src/internal/blockchain/block/Types.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
```

Comment:
We must be having more than 40 different headers with the same name now - why are we tolerating such nonsense. The name so used becomes useless.

Reply:
MT-85

Reply:
There is no TODO note so there will be no way to get back to this issue once this review is closed.

Reply:
isnow :)